src/DacpacTool/Extensions.cs

```diff
@@ -204,17 +204,13 @@ public static IEnumerable<string> GetReferencedDacPackages(this TSqlModel model)
             return result;
         }
 
-        public static void AddSqlCmdVariables(this TSqlModel model, string[] variableNames)
+        public static void AddSqlCmdVariable(this TSqlModel model, string variableName, string variableDefaultValue)
         {
             var dataSchemaModel = GetDataSchemaModel(model);
 
             var customData = Activator.CreateInstance(Type.GetType("Microsoft.Data.Tools.Schema.SchemaModel.CustomSchemaData, Microsoft.Data.Tools.Schema.Sql"), "SqlCmdVariables", "SqlCmdVariable");
-
-            foreach (var variableName in variableNames)
-            {
-                var setMetadataMethod = customData.GetType().GetMethod("SetMetadata", BindingFlags.Public | BindingFlags.Instance);
-                setMetadataMethod.Invoke(customData, new object[] { variableName, string.Empty });
-            }
+            var setMetadataMethod = customData.GetType().GetMethod("SetMetadata", BindingFlags.Public | BindingFlags.Instance);
```

Comment:
Now that I look at this I'm wondering if this works the way we expect it to work. Previously we would only add one `CustomSchemaData` object to the model with all the variables. Now we're adding it multiple times for each variable. To be honest I don't know if that matters, but I think we should have a look at the resulting `.dacpac` from both a `.sqlproj` and an `MSBuild.Sdk.SqlProj` project to make sure we're producing the correct output.

Reply:
Just found out issue #20 that was about this problem, so I reverted the commit and moved console logging inside `Extensions` as you were suggesting.